newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -277,6 +281,36 @@ export default class AuthenticatedUserProvider extends React.Component<
         console.error('Error while loading user cloud projects:', error);
       }
     );
+    listUserPurchases(authentication.getAuthorizationHeader, {
+      userId: firebaseUser.uid,
+      role: 'receiver',
+      productType: 'asset-pack',
+    }).then(
+      purchases =>
+        this.setState(({ authenticatedUser }) => ({
+          authenticatedUser: {
+            ...authenticatedUser,
+            ownedAssetPackIds: purchases.map(purchase => purchase.productId),
+          },
+        })),
+      error => {
+        console.error('Error while loading user purchases:', error);
+      }
+    );
+    listReceivedAssetShortHeaders(authentication.getAuthorizationHeader, {
+      userId: firebaseUser.uid,
+    }).then(
+      assetShortHeaders =>
+        this.setState(({ authenticatedUser }) => ({
+          authenticatedUser: {
+            ...authenticatedUser,
+            ownedAssetShortHeaders: assetShortHeaders,
```

Comment:
The method is called `received`, but the resulting assetShortHeaders are stored into something "owned".
So we must chose to be consistent:
- is this owned?
- or is this received?